weblate_web/templates/snippets/service.html

```diff
@@ -1,107 +1,167 @@
 {% load i18n %}
+{% load static %}
+{% load date_diff %}
 
-            <div class="payment-form">
-                <div class="form-line">
-                    <div class="line-left">{% trans "Service" %}</div>
+<div class="payment-form pt-pb">
+    <div class="form-line style6 bdr-btm">
+        <div class="service-variant">
+            <p>{{ service.get_status_display }}</p>
+        <div class="service-time">
+            {% with subscriptions=service.subscription_set.all %}
+            {% for subscription in subscriptions %}
+            {% if subscription.expires|days_diff_from_today < 30 %}
```

Comment:
Probably better to calculate the diff just once and reuse the calculation instead of doing that several times:

```suggestion
            {% with days_from_today=subscription.expires|days_diff_from_today %}
            {% if days_from_today < 30 %}
```

Reply:
Yes, you are right, i have implemented this practice.